keyboards/ergodox_infinity/keymaps/nordic_ergo/keymap.c

```diff
@@ -1,24 +1,16 @@
 // Nordic layout for Ergodox infinity
-
 #include QMK_KEYBOARD_H
-#include "debug.h"
 #include "action_layer.h"
 #include "version.h"
 #include "keymap_nordic.h"
 #include "keymap_german.h"
 
-// Layer names
-#define BASE 0 // default layer
-#define FUNCL 1 // function layer
-#define SYMB 2 // symbol layer
-#define MDIA 3 // media keys
+#define BASE 0   // default layer
+#define FUNCL 1  // function layer
+#define SYMB 2   // symbol layer
+#define MDIA 3   // media keys
```

Comment:
Using an `enum` is recommended here.

```c
enum layer_names {
    BASE,   // default layer
    FUNCL,  // function layer
    SYMB,   // symbol layer
    MDIA    // media keys
};
```

This is the same as the defines that are already here.